### PR TITLE
Static analysis issues fixes

### PR DESCRIPTION
**Pull request by dmitrybrazhkin**
Signed-off-by: Dmitry Brazhkin <dmitry.brazhkin@intel.com>

### CODE REVIEW

File: _studio/shared/umc/codec/h264_dec/src/umc_h264_mfx_supplier.cpp [hunk 1]
```diff
@@ -388,7 +388,7 @@ Status MFXTaskSupplier::DecodeSEI(NalUnit *nalUnit)
         bitStream.Reset((uint8_t*)swappedMem.GetPointer(), (uint32_t)swappedMem.GetDataSize());
 
         NAL_Unit_Type nal_unit_type;
-        uint32_t nal_ref_idc;
+        uint32_t nal_ref_idc = 0;
```
Comment (dvrogozh) [inline _studio/shared/umc/codec/h264_dec/src/umc_h264_mfx_supplier.cpp]: Why we don't initialize nal_unit_type then? And what we actually get in the bitStream.GetNALUnitType(nal_unit_type, nal_ref_idc); - both values or only one?

File: _studio/mfx_lib/decode/h265/src/mfx_h265_dec_decode.cpp [hunk 2]
```diff
@@ -786,7 +786,11 @@ mfxStatus VideoDECODEH265::GetDecodeStat(mfxDecodeStat *stat)
     m_stat.NumSkippedFrame = m_pH265VideoDecoder->GetSkipInfo().numberOfSkippedFrames;
     m_stat.NumCachedFrame = 0;
 
-    H265DecoderFrame *pFrame = m_pH265VideoDecoder->GetDPBList()->head();
+    H265DBPList *dpb = m_pH265VideoDecoder->GetDPBList();
+    if (!dpb)
+        return MFX_ERR_NULL_PTR;
```
Comment (dmitry-gurulev) [inline _studio/mfx_lib/decode/h265/src/mfx_h265_dec_decode.cpp]: `MFX_ERR_NULL_PTR` is described as 
> NULL pointer in the input or output arguments
Having null pointer for DPB here looks to be unexpected, so I think `MFX_ERR_UNDEFINED_BEHAVIOR` status would be better.

Comment (dmitrybrazhkin) [reply _studio/mfx_lib/decode/h265/src/mfx_h265_dec_decode.cpp]: Fixed

File: _studio/shared/umc/codec/h264_dec/src/umc_h264_mfx_supplier.cpp (hunk 1 above)
Comment (dvrogozh) [reply _studio/shared/umc/codec/h264_dec/src/umc_h264_mfx_supplier.cpp]: I would either initialize both or have error check for GetNALUnitType.

Comment (dmitrybrazhkin) [reply _studio/shared/umc/codec/h264_dec/src/umc_h264_mfx_supplier.cpp]: Done

**Pull request merged**